src/main/java/de/tum/in/www1/artemis/config/migration/setups/localvc/LocalVCMigrationEntry.java

```diff
@@ -0,0 +1,26 @@
+package de.tum.in.www1.artemis.config.migration.setups.localvc;
+
+import java.net.URL;
+
+import org.springframework.beans.factory.annotation.Value;
+
+import de.tum.in.www1.artemis.config.migration.setups.ProgrammingExerciseMigrationEntry;
+import de.tum.in.www1.artemis.repository.AuxiliaryRepositoryRepository;
+import de.tum.in.www1.artemis.repository.ProgrammingExerciseRepository;
+import de.tum.in.www1.artemis.repository.ProgrammingExerciseStudentParticipationRepository;
+import de.tum.in.www1.artemis.repository.SolutionProgrammingExerciseParticipationRepository;
+import de.tum.in.www1.artemis.repository.TemplateProgrammingExerciseParticipationRepository;
+
+public abstract class LocalVCMigrationEntry extends ProgrammingExerciseMigrationEntry {
```

Comment:
Why do we need this super class in between?
Maybe add a comment above the class explaining that

Reply:
:heavy_check_mark: I added a JavaDoc comment with cf570accff25731b402fb297969b772067374758